### PR TITLE
Header, search-box component

### PR DESCRIPTION
**Pull request by ArturKragel**

### CODE REVIEW

File: src/components/Header/header.styled.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+export const Header = styled.header`
+  background-color: #282c34;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
```
Comment (viktoryia-arabei) [inline src/components/Header/header.styled.jsx:9]: why not just use 20px?

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,29 @@
...
+
+
+
 function App() {
+    const [movie, setMovie] = useState([]);
+
+    useEffect( async()=>{
+        const resp = await axios.get('http://www.omdbapi.com/?t=Men&apikey=924bf6');
+        console.log(resp.data)
+    }, []);
```
Comment (viktoryia-arabei) [inline src/App.js:17]: let's use redux

File: src/components/Header/header.styled.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+export const Header = styled.header`
+  background-color: #282c34;
```
[the remark this answers is not in the document]
Comment (ArturKragel) [reply src/components/Header/header.styled.jsx:4]: done at the next pull request

File: src/App.js (hunk 2 above)
Comment (ArturKragel) [reply src/App.js:17]: In process

**Pull request merged**